src/docs/concepts/advanced/authoring_component_libraries.md

```diff
@@ -0,0 +1,255 @@
+---
+title: Authoring component libraries
+weight: 7
```

Comment:
The `weight` in the frontmatter is used to define the orders of the articles in the same folder. Lower weight goes first (higher up).